Tiltfile

```diff
@@ -40,10 +43,14 @@ def web_lint():
   ts_deps = get_all_ts_files("web")
   local_resource("web_lint", "cd web && yarn run check", deps=ts_deps, resource_deps=["yarn_install"])
 
+def go_vendor():
+  local_resource("go_vendor", "make vendor", deps=['go.sum', 'go.mod'])
```

Comment:
not really related to this PR, so feel free to merge anyway, but made me wonder - why filter out vendor in get_all_go_files? This seems handy:
1. I import something in a go file that's not in vendor
2. Tilt runs go build, which fails because it's not in go vendor, but maybe it automatically updates go.mod (not sure if it does this with -mod vendor)? or maybe I run go get to add it to go.mod
3. Tilt runs make vendor to update vendor
4. Tilt re-runs go build now that vendor has the library

Reply:
Good point!